ledger/evalindexer.go

```diff
@@ -179,7 +180,7 @@ func (l indexerLedgerConnector) GenesisHash() crypto.Digest {
 }
 
 // GenesisProto is part of LedgerForEvaluator interface.
-func (l indexerLedgerConnector) GenesisProto() config.ConsensusParams {
```

Comment:
this would require to update the indexer that is not great. Can we add `GenesisProtoVersion` function instead and preserve the old one with its signature?

Reply:
haven't we talked about not changing the signature of `ledger.GenesisProto`?